binding.go

```diff
@@ -13,11 +13,22 @@ import (
 	"github.com/flamego/flamego"
 )
 
+// Options is a struct for specifying configuration options for the binding middleware.
+type Options struct {
+	// CustomErrorHandler will be invoked if errors occurred.
+	CustomErrorHandler func(flamego.Context, Errors)
```

Comment:
```suggestion
	// ErrorHandler will be invoked when errors occurred.
	ErrorHandler func(flamego.Context, Errors)
```

We may use this field to define default error handler.